server/jetty-app/README.md

```diff
@@ -1,76 +1,40 @@
-This document is oriented towards getting a server up for local development.
-If you are interested in running a native production release, please see
-[https://deephaven.io/core/docs/how-to-guides/configuration/native-application/](https://deephaven.io/core/docs/how-to-guides/configuration/native-application/).
+If you wish to use Deephaven from a production environment, which is simpler but source code cannot be modified, see one of the following documents:
 
-### Groovy Quickstart
+- [Quickstart for Docker](https://deephaven.io/core/docs/tutorials/quickstart/)
+- [How to configure the Deephaven native application](https://deephaven.io/core/docs/how-to-guides/configuration/native-application/)
 
-```shell
-./gradlew server-jetty-app:run -Pgroovy 
-```
-
-### Python Quickstart
-
-1. Setup virtual environment:
-
-   ```shell
-   python -m venv /tmp/my-dh-venv
-   source /tmp/my-dh-venv/bin/activate
-   ```
-
-1. Build and install wheel
-
-   ```shell
-   ./gradlew py-server:assemble
-
-   # replace with the appropriate <version>
-   pip install "py/server/build/wheel/deephaven_core-<version>-py3-none-any.whl[autocomplete]"
-
-   # To install without the optional `autocomplete` feature, run:
-   # pip install "py/server/build/wheel/deephaven_core-<version>-py3-none-any.whl"
-   ```
-
-1. Run
+This README deals with general development for either the Python or Groovy server-side API. For Python-specific development instructions, see the [Python development README](../../py/README.md).
 
-   ```shell
-   ./gradlew server-jetty-app:run
-   ```
+## Local development
 
-**Note:**
+`./gradlew server-jetty-app:run` will incorporate local Java changes on each start. If you are not frequently changing Java code, see the next section.
 
-* This is not an exhaustive guide to managing python environments
-* Depending on your OS and how your PATH is setup, you may need to use `python3`, or a path to the explicit python version you want to use
-* You may choose to setup a "permanent" virtual environment location
-* You'll need to re-install the wheel anytime you are making python code changes that affect the wheel
-* `pip` can be a pain if you are trying to (re-)install a wheel with the same version number as before
-  * A `pip install --force-reinstall --no-deps "py/server/build/wheel/deephaven_core-<version>-py3-none-any.whl[autocomplete]"` may do the trick
-* You can install other python packages in your venv using `pip install <some-other-package>`
-* You can setup multiple virtual environments, and switch between them as necessary using `source /path/to/other-venv/bin/activate`
-* You can de-activate the virtual environment by running `deactivate`
-* You can use the `VIRTUAL_ENV` environment variable instead of sourcing / activating virtual environments: `VIRTUAL_ENV=/my/venv ./gradlew server-jetty-app:run`
+```shell
+./gradlew server-jetty-app:run # Python session (default)
+./gradlew server-jetty-app:run -Pdebug # Attach a debugger to the Python session
```

Comment:
I'm pretty sure this attaches the Java debugger for server Java code. The language for the code studio doesn't matter

Reply:
Yep this is a shorthand for attaching the JVM args [here](https://github.com/deephaven/deephaven-core/blob/main/server/jetty-app/build.gradle#L79). Attaches Java debugger to port 5005